### PR TITLE
Update stations_name.json

### PR DESCRIPTION
**Pull request by Nickduino**
Added a few new stations (2 or three) and corrected the names and addresses of a few more.

One Auchan station has its address completely wrong on the government's website (Route d'issoire instead of avenue du Roussillon), I know the station and I quadruple-checked: I am right.

image

### CODE REVIEW

File: custom_components/prix_carburant/stations_name.json [hunk 1]
```diff
@@ -32301,15 +32324,15 @@
...
+    "name": "Carrefour MONTEssoN",
     "brand": "Carrefour"
   },
   "78360002": {
     "name": "Carrefour Market",
     "brand": "Carrefour Market"
   },
   "78360004": {
-    "name": "RELAIS DE MONTESSON",
+    "name": "RELAIS DE MONTEssoN",
```
Comment (Aohzan) [inline custom_components/prix_carburant/stations_name.json:32335]: le remplacement n'est pas correct partout

File: custom_components/prix_carburant/stations_name.json [hunk 2]
```diff
@@ -32301,15 +32324,15 @@
     "brand": "Total Access"
   },
   "78360001": {
-    "name": "Carrefour MONTESSON",
+    "name": "Carrefour MONTEssoN",
```
Comment (Aohzan) [inline custom_components/prix_carburant/stations_name.json:32327]: le remplacement n'est pas correct partout

File: custom_components/prix_carburant/stations_name.json [hunk 3]
```diff
@@ -26085,7 +26108,7 @@
     "brand": "Système U"
   },
   "64480004": {
-    "name": "EOR LARRESSORE ETCHEGARAY",
+    "name": "EOR LARREssoRE ETCHEGARAY",
```
Comment (Aohzan) [inline custom_components/prix_carburant/stations_name.json:26111]: le remplacement n'est pas correct partout

**Review by Aohzan — CHANGES_REQUESTED** [review]
there is other to fix

**Comment by Nickduino** [comment]
Merde.

Je vérifie et corrige tous les *esso* de suite

**Comment by Nickduino** [comment]
C'est fait

**Review by Aohzan — APPROVED** [review]

**Pull request merged**